packages/frontend/vite.config.ts

```diff
@@ -20,7 +38,21 @@ export default defineConfig({
             ],
         },
     },
+
+    // Uncomment this if you are using workers.
+    // worker: {
+    //  plugins: [
+    //    viteTsConfigPaths({
+    //      root: '../../',
+    //    }),
+    //  ],
+    // },
+
     test: {
+        cache: {
+            dir: "../../node_modules/.vitest",
+        },
+        include: ["src/**/*.{test,spec}.{js,mjs,cjs,ts,mts,cts,jsx,tsx}"],
```

Comment:
Why is this include now explicitly needed?

Reply:
by default it looks for `['**/__tests__/**/*.?(c|m)[jt]s?(x)', '**/?(*.){test,spec}.?(c|m)[jt]s?(x)']`, so I guess we can remove it